packages/tbds-support/lib/breakpoints.scss

```diff
@@ -0,0 +1,15 @@
+$tbds-breakpoint-small: 480px;
```

Comment:
Thoughts on using `em`-based media queries? As far as I know, they are still the most robust unit to use.

More info: https://zellwk.com/blog/media-query-units/